Mathlib/Data/Set/Card.lean

```diff
@@ -180,6 +180,13 @@ theorem encard_prod {s : Set α} {t : Set β} : (s ×ˢ t).encard = s.encard * t
   unfold encard
   simp [ENat.card_congr (Equiv.Set.prod ..)]
 
+@[simp]
+theorem encard_pi_eq_prod_encard [h : Fintype α] {ι : α → Type*} {s : ∀ i : α, Set (ι i)} :
+  (Set.pi Set.univ s).encard = ∏ i, (s i).encard := by
```

Comment:
It would be nice to have a partial version of this, ie `(Set.pi t s).encard = ∏ i \in t, (s i).encard`

Reply:
It's not quite general enough in practice, but I agree it's tough to modify, so let's leave it as is for the moment.